ChopChop/Models/OnlineRecipes/OnlineRecipe.swift

```diff
@@ -1,24 +1,25 @@
 import Foundation
-import FirebaseFirestoreSwift
 
-class OnlineRecipe: Identifiable {
-    private(set) var id: String
-    private(set) var userId: String
-    private(set) var parentOnlineRecipeId: String?
-
-    private(set) var name: String
-    private(set) var servings: Double
-    private(set) var cuisine: String?
-    private(set) var difficulty: Difficulty?
-    private(set) var stepGraph: RecipeStepGraph
-    private(set) var ingredients: [RecipeIngredient]
-    private(set) var ratings: [RecipeRating]
-    private(set) var created: Date
+struct OnlineRecipe: Identifiable, CachableEntity {
```

Comment:
Hmm why is it a struct now? Is it so that it can conform to cachableEntity? And i checked our report say OnlineRecipe is a class xD

Reply:
lol uh just change ah, no point making into a class